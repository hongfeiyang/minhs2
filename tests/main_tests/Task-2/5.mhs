main = (recfun f :: (Bool -> Int -> Bool) x y = y);
