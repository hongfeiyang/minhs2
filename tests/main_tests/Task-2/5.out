main :: (Bool -> Int -> Bool) = (recfun f :: (Bool -> Int -> Bool) x y = y);